src/soc/ibm/power9/romstage.c

```diff
@@ -363,7 +363,7 @@ static void build_mvpds(uint8_t chips)
 {
 	uint8_t chip;
 
-	printk(BIOS_EMERG, "Building MVPDs...\n");
+	printk(BIOS_INFO, "Building MVPDs...\n");
```

Comment:
`BIOS_NOTICE`, this takes very long time

Reply:
Ok, pushed a change with this.